templates/thirdparty/qtpromise/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ include(FetchContent)
 
 FetchContent_Declare(qtpromise
   GIT_REPOSITORY https://github.com/simonbrunel/qtpromise.git
-  GIT_TAG v0.6.0
+  GIT_TAG v0.7.0
```

Comment:
Should we create an issue to move to Qt's own promise at least for tracking why it wasn't done?

Reply:
https://github.com/apigear-io/template-qtcpp/issues/49 created